src/Symfony/Component/Console/Tests/Fixtures/CustomCommandResolver.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Tests\Fixtures;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\CommandsResolver\CommandResolverInterface;
+
+/**
+ * @author Ivan Shcherbak <dev@funivan.com>
+ */
+class CustomCommandResolver implements CommandResolverInterface
```

Comment:
I want to show how to implement `CommandResolverInterface` from zero.
What do you mean: `add default fixture code in constructor`? Can you show me an example, I don`t understand.